templates/macros/actions.html.twig

```diff
@@ -211,6 +226,10 @@
             {% set actions = actions|merge({'edit': {'url': path('timesheet_edit', {'id': timesheet.id}), 'class': class}}) %}
         {% endif %}
 
+        {% if timesheet.end and is_granted('start', timesheet) %}
```

Comment:
I like the shorter if, but I would move this back above edit, as edit is already available via the table row click

Reply:
OK